server/apps/users/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path, re_path
+from server.apps.users import views
+
+app_name = "servers.apps.users"
+
+urlpatterns = [
+    path("profile/", views.user_profile, name="profile"),
+    path("greetings/", views.user_profile, {"first_visit": True}, name="greetings"),
```

Comment:
This might be a left-over from testing? 🙂

Reply:
This is the fix I added to avoid using `UserProfile` existence as a "first visit" condition. The OpenHumans redirect page is now the greetings page, which happens to be the profile page + greeting message.

This is definitely preferable to the approach used earlier, because it avoids writing on `GET` of the profile page, which I understand is generally frowned upon.

Reply:
Ah, got it, but does that mean that the `login_user` in the `main.views` is now unused and that by default no `UserProfile` is created when logging in?

Reply:
Ah, no, that's my mistake in how I described it (sorry). To correct what I wrote: `login_user` is the entry point. That then either redirects to `greetings` or `overview` depending on whether it's the first visit or not.

Reply:
No worries at all! Web stuff is always such a complex beast! :D 

Last double check that I got it, the flow is: `login_user`, which creates the `UserProfile` and then we move the user to `greetings`?

Reply:
That sounds correct to me, yes (the [relevant code](https://github.com/llewelld/AutSPACEs/blob/9782fbe132e8c8fbcce26bd914edc1fc115875d9/server/apps/main/views.py#L133)).

Reply:
Yep, that's great! Thanks for walking me through it 🙏

Reply:
(just noticed: I will fix the spelling in that docstring!)